packages/as-proto-gen/src/generate/field.ts

```diff
@@ -309,31 +311,26 @@ export function generateFieldTypeInstruction(
   }
 }
 
+export function isNullableFieldType(
+  fieldDescriptor: FieldDescriptorProto
+): boolean {
+  const fieldType = fieldDescriptor.getType();
+  assert.ok(fieldType !== undefined);
+
+  return fieldType === Type.TYPE_MESSAGE;
```

Comment:
Shouldn't this always be false, given that there is no null in protobuf3? It is not possible to distinguish from a default message vs a "null" one AFAIK.